docs/data/date-pickers/timezone/DayjsTimezone.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+export default function DayjsTimezone() {
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <DemoContainer components={['DateTimePicker']}>
+        <DateTimePicker
+          timezone="America/New_York"
+          defaultValue={dayjs.tz('2022-04-17T15:30', 'America/New_York')}
```

Comment:
I'm wondering what would append if you pass just `dayjs()`

Might be interesting for two reasons:
- If the code fails with error message: copy pasting the error code in Google could link to the page
- If the code does not fails: would worth adding a warning about behavior

---
I don't know why, I did not manage to run one of the demo in the playground whereas it works in the docs page 🙈

Reply:
`days()` will use the locale timezone, so you should see the converted value on screen.
For example if you are in France and you set `timezone="Armerica/New_York"`, `dayjs()` at 8AM should render something like 2AM in your field.